.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           required-ros-distributions: foxy
```

Comment:
Not sure about what this does, but should it be galactic?

Reply:
```suggestion
          required-ros-distributions: galactic
```